src/buffer_info.cpp

```diff
@@ -148,4 +148,29 @@ BuffInfo::BuffInfo(MemLayout_t layout, int nk, int c, int h, int w, int g, int _
     }
 }
 
+MultiBufferWorkspaceTraits::MultiBufferWorkspaceTraits(std::initializer_list<size_t> v_size_,
+                                                       size_t alignment_)
+    : v_size(v_size_), alignment(alignment_)
+{
+    buffer_num         = v_size.size();
+    size_t each_offset = 0;
+    v_offset.push_back(each_offset);
+    for(auto each_size : v_size)
+    {
+        size_t padding = (alignment - (each_size % alignment)) % alignment;
+        each_offset += each_size + padding;
+        v_offset.push_back(each_offset);
+    }
+    size = v_offset[buffer_num];
+}
+
+size_t MultiBufferWorkspaceTraits::GetSize() const { return size; }
```

Comment:
I recommend removing `size` from the class and returning `v_offset.back()`.

Reply:
Ok. changed.